Sources/Mustache/ContentType.swift

```diff
@@ -36,21 +38,32 @@ struct HTMLContentType: MustacheContentType {
 ///
 /// The string is read from the "CONTENT_TYPE" pragma `{{% CONTENT_TYPE: type}}`. Replace type with
 /// the content type required. The default available types are `TEXT` and `HTML`. You can register your own
-/// with `MustacheContentTypes.register`.
-public enum MustacheContentTypes {
-    static func get(_ name: String) -> MustacheContentType? {
-        self.types[name]
+/// with `MustacheContentTypes.shared.register`.
+public class MustacheContentTypes: @unchecked Sendable {
```

Comment:
This is a breaking change. You need to revert back to the enum. See comment about `nonisolated(unsafe)`

Reply:
Thank you for your suggestion. `swift package diagnose-api-breaking-changes` is very useful. I never knew that.